### ISSUE TITLE
Android onLogin not triggered but reload the app

### ISSUE THREAD

**Issue by leyuan** [magus/react-native-facebook-login#240]
### Description
It was working all good on IOS, but when I tried to play around with it on Android, I found the component is not triggering the `onLogin` callback. Instead, it seems to reload the app, and calls `onLoginNotFound`.

And as you can see in the gif below, after Facebook shows Confirm Login window, things started to working, but after clicking `Continue` it reloads the app.

### Code
```
      <FBLogin style={{ marginBottom: 10, alignSelf: 'center'}}
        permissions={permissions}
        loginBehavior={FBLoginManager.LoginBehaviors.Native}
        onLogin={this.onLogin.bind(this)}
        onLogout={this.onLogout.bind(this)}
        onLoginFound={this.onLoginFound.bind(this)}
        onLoginNotFound={this.onLoginNotFound.bind(this)}
      />
```
Basically just console logs in the callbacks.

### Env
```
    "react": "16.0.0-alpha.3",
    "react-native": "^0.43.4",
```

### Screenshots
![fb-login](https://cloud.githubusercontent.com/assets/1417993/25922163/52d5ebe2-3595-11e7-86cb-2a2de3cfe1de.gif)
